packages/pango.rb

```diff
@@ -11,6 +11,7 @@ class Pango < Package
   depends_on 'fontconfig'
```

Comment:
It is outside of this PR.  I don't want to optimize acyclic paths of dependencies to avoid causing new problem by this PR.